### DISCUSSION TITLE
Open another Database with a Password from another Database

### DISCUSSION THREAD

**Discussion by mal-tee** [keepassxreboot/keepassxc#9279] [Ideas]
I have a personal KeePassXC Database and a shared one. I store the password for the shared database in my personal database. If I want to open the shared file, I usually copy the password, change to the different tab inside KeePassXC and paste it into the passphrase field.

Is it already possible to automate this? Otherwise this might be a convenient feature? Or could this be a problem for security?

**Comment by droidmonkey** [comment]
Check out the user guide, section is auto open databases.

**Reply by mal-tee** [reply]
Thank you!
